apps/storybook/src/ComboBox.test.ts

```diff
@@ -27,6 +28,12 @@ describe('ComboBox', () => {
           items[3].click();
         });
       }
+
+      // Hide images if present to avoid uncertainties in testing
+      if (testName === 'Disabled') {
+        cy.get('img.iui-icon').hide();
```

Comment:
it should also work if we don't target the class here. will be easier to maintain if we change the class in the future.
```suggestion
        cy.get('img').hide();
```

Reply:
**[DONE]** I had actually tried to make the selectors specific since I noticed `Tile.test.ts` [L30](https://github.com/iTwin/iTwinUI-react/blob/main/apps/storybook/src/Tile.test.ts#L30) also selected by class name.

But I guess that was because there was no `<img>` tag there. The image was in the `<div>` tag itself and so needed the class name selector.